pypeman/remoteadmin.py

```diff
@@ -140,19 +140,34 @@ def get_channel(self, name):
             'status': channels.BaseChannel.status_id_to_str(chan.status)
         }
 
-    async def list_msg(self, channel, start=0, count=10, order_by='timestamp'):
+    async def list_msg(self, channel, start=0, count=10, order_by='timestamp', mk_b64pickle=False):
         """
         List first `count` messages from message store of specified channel.
 
         :params channel: The channel name.
+        :param mk_b64pickle: if True (yield payload / ctx fields as b64
+                encoded pickles)
+                if False a a dict with a field type and a field repr will be
```

Comment:
`a a dict`

Reply:
will do

Reply:
done in d02100f8d8d1d20c677741ada689b0b016f27f2c

Reply:
can close?